docs/build/index_template.md

```diff
@@ -0,0 +1,5 @@
+# Operation Index
+
+{% for op in operations -%}
```

Comment:
Not quite clear to me what these `for` loops and things like that are here for. Perhaps we can clarify in the call today. Perhaps it's Jinja magic?